### PR TITLE
Add links for Portworx plugin support

### PR DESCRIPTION
**Pull request by disrani-px**

### CODE REVIEW

**Comment by disrani-px** [comment]
cc @rosskukulinski

**Comment by skriss** [comment]
LGTM

File: docs/support-matrix.md [hunk 1]
```diff
@@ -49,3 +49,5 @@ After you publish your plugin, open a PR that adds your plugin to the appropriat
 [10]: https://kubernetes.slack.com/messages/ark-dr
 [11]: https://github.com/heptio/ark/issues
 [12]: https://github.com/aws/aws-sdk-go/aws
+[13]: https://slack.portworx.com
```
Comment (nrb) [inline docs/support-matrix.md]: Is there a specific channel that would be best?

**Comment by nrb** [comment]
Thanks for the PR @disrani-px! I just have 1 question, otherwise it looks good.

**Comment by rosskukulinski** [comment]
Thanks @disrani-px. I second Nolan's question but +1

File: docs/support-matrix.md (hunk 1 above)
Comment (disrani-px) [reply docs/support-matrix.md]: There is no specific channel for ark, but we have a channel for kubernetes. I can link that there.

**Pull request merged**